File: Python/2037.py
class Solution(object):
    def minMovesToSeat(self, seats, students):
        """
        :type seats: List[int]
        :type students: List[int]
        :rtype: int
        """
        
        seats.sort()
        students.sort()
        n = len(students) 
        moves = 0
    
        for i in range(n):
            moves += abs(students[i]-seats[i])
            
        return moves
        



    
            

File: Java/2037.java
import java.util.Arrays;

class Solution {
    public int minMovesToSeat(int[] seats, int[] students) {


        int a[] = Arrays.sort(seats);
        int b[] = Arrays.sort(students);
        int moves = 0; 

        for(int i = 0; i < students.length; i ++){
            moves = moves + Math.abs(a[i] - b[i]); 
        }

        return moves; 

    }
}